src/charm.py

```diff
@@ -20,6 +20,39 @@
 DEFAULT_METRICS_ENDPOINT_PORT = 80
```

Comment:
Not sure if this default is:

1. correct: I haven't seen any port 80 metrics endpoints so far.
2. needed: if port is omitted, 80 is the default anyway, no?

Would it be so bad if we require the user to provide it explicitly?

Reply:
There really is no standard, but `80` is the default web port :-)

Reply:
Prometheus got it covered already:
```
<host>: a valid string consisting of a hostname or IP followed by an optional port number
```
I think we better off dropping it.